backend/src/main/java/gov/cdc/usds/simplereport/service/TenantDataAccessService.java

```diff
@@ -0,0 +1,91 @@
+package gov.cdc.usds.simplereport.service;
+
+import gov.cdc.usds.simplereport.api.CurrentTenantDataAccessContextHolder;
+import gov.cdc.usds.simplereport.config.AuthorizationProperties;
+import gov.cdc.usds.simplereport.config.authorization.OrganizationExtractor;
+import gov.cdc.usds.simplereport.config.authorization.OrganizationRole;
+import gov.cdc.usds.simplereport.config.authorization.OrganizationRoleClaims;
+import gov.cdc.usds.simplereport.db.model.ApiUser;
+import gov.cdc.usds.simplereport.db.model.Organization;
+import gov.cdc.usds.simplereport.db.model.TenantDataAccess;
+import gov.cdc.usds.simplereport.db.model.auxiliary.PermissionsData;
+import gov.cdc.usds.simplereport.db.repository.TenantDataAccessRepository;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Date;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional(readOnly = false)
+public class TenantDataAccessService {
+
+  private static final int VALID_MINUTES = 60;
+
+  private static final Logger LOG = LoggerFactory.getLogger(TenantDataAccessService.class);
+
+  @Autowired private TenantDataAccessRepository _repo;
+  @Autowired private CurrentTenantDataAccessContextHolder _contextHolder;
+  @Autowired private AuthorizationProperties _authProperties;
+  @Autowired private OrganizationExtractor _extractor;
+
+  public Optional<Set<String>> getTenantDataAccessAuthorityNames(ApiUser apiUser) {
```

Comment:
It doesn't really.  The only thing it would differentiate is between not having a tenant data access record vs a record that contains no authorities.  The latter case isn't something that makes sense and isn't currently possible without modifying the database record outside of the app.  Agree that this is a good change.